### PR TITLE
Fix broken hearts

### PR DESCRIPTION
**Pull request by sarahikeda**
Small PR, full of <3 and :). I don't think it was tokenizing the heart, prior. See what you think! @johnnytomcat

### CODE REVIEW

**Comment by johnnytomcat** [comment]
Hi @sarahikeda, 
I no longer have access to update this repo, but I am maintaining my own fork so feel free to create the PR against that and I can merge it in.

Thanks for the fix!

**Comment by eponvert** [comment]
@johnnytomcat you're a collaborator once again :smile_cat:

File: lib/rb_lib_text/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module RbLibText
-  VERSION = "0.0.1"
+  VERSION = "0.0.2"
```
Comment (eponvert) [inline lib/rb_lib_text/version.rb:2]: @johnnytomcat @sarahikeda sorry, is it possible to have like a SNAPSHOT version, or is that not a Ruby lib thing?

File: lib/rb_lib_text.rb [hunk 2]
```diff
@@ -14,23 +14,24 @@ def self.or_pattern
         tags_contractions: '[\w]+[\'‘’][\w]+', #don't split don't and can't and it's
         emails: '[\w\.\d]+@[\w\.\d]+\.[\w]+', #catch email addresses
         urls: 'https?://[-_/~%\w\d\.]*[_/~\w\d]', #Catch url addresses
-        #sideways_text_emoji: '>?[:;=][\'\-D\)\]\(\[pPdoO/\*3\\]+',
+        # sideways_text_emoji: '>?[:;=][\'\-D\)\]\(\[pPdoO/\*3\\]+',
```
Comment (eponvert) [inline lib/rb_lib_text.rb:17]: can we removed the commented code?

File: spec/rb_lib_text_spec.rb [hunk 3]
```diff
@@ -10,4 +10,19 @@
     tokens = RbLibText.tokens(sentance)
     expect(tokens.count).to eq(16)
   end
+
+  it 'keeps hearts intact' do
+    sentence = "My first open source PR! <3 #itwasfun"
+    tokens = RbLibText.tokens(sentence)
+    expect(tokens).to eq(["My", "first", "open", "source", "PR", "!", "<3", "#itwasfun"])
+    expect(tokens.count).to eq(8)
```
Comment (eponvert) [inline spec/rb_lib_text_spec.rb:18]: you shouldn't need to check the length of the list after you check its contents, should you?

File: spec/rb_lib_text_spec.rb [hunk 4]
```diff
@@ -10,4 +10,19 @@
...
+  end
+
+  it 'recognizes smiley faces' do
+    sentence = ":)\n;)\n=)\n:'-(\n>:3\n:D\n:P\n8)\n:-*\n:-/\n:O\n:p\n:("
+    tokens = RbLibText.tokens(sentence)
+    expect(tokens).to eq([":)", ";)", "=)", ":'-(", ">:3", ":D", ":P", "8)", ":-*", ":-/", ":O", ":p", ":("])
+    expect(tokens.count).to eq(13)
```
Comment (eponvert) [inline spec/rb_lib_text_spec.rb:25]: ditto ^^^

**Comment by eponvert** [comment]
Thanks very much for the unit tests! :100:

**Comment by johnnytomcat** [comment]
Move to a different PR

**Comment by eponvert** [comment]
@johnnytomcat huh? why?

**Comment by johnnytomcat** [comment]
It was already merged in my fork, i just brought this one up to date with mine.

**Comment by eponvert** [comment]
OK, cool, thanks!

**Pull request closed without merging**